tests/e2e/dualstack/dualstack_test.go

```diff
@@ -159,10 +159,18 @@ values:
 						Expect(k.Label("namespace", IPv6Namespace, "istio-injection", "enabled")).To(Succeed(), "Error labeling ipv6 namespace")
 						Expect(k.Label("namespace", SleepNamespace, "istio-injection", "enabled")).To(Succeed(), "Error labeling sleep namespace")
 
-						Expect(k.WithNamespace(DualStackNamespace).Apply(common.GetSampleYAML(version, "tcp-echo-dual-stack"))).To(Succeed(), "error deploying tcpDualStack pod")
-						Expect(k.WithNamespace(IPv4Namespace).Apply(common.GetSampleYAML(version, "tcp-echo-ipv4"))).To(Succeed(), "error deploying ipv4 pod")
-						Expect(k.WithNamespace(IPv6Namespace).Apply(common.GetSampleYAML(version, "tcp-echo-ipv6"))).To(Succeed(), "error deploying ipv6 pod")
-						Expect(k.WithNamespace(SleepNamespace).Apply(common.GetSampleYAML(version, "sleep"))).To(Succeed(), "error deploying sleep pod")
+						Expect(k.WithNamespace(DualStackNamespace).
+							ApplyKustomize("tcp-echo-dual-stack")).
+							To(Succeed(), "error deploying tcpDualStack pod")
+						Expect(k.WithNamespace(IPv4Namespace).
+							ApplyKustomize("tcp-echo-ipv4")).
+							To(Succeed(), "error deploying ipv4 pod")
+						Expect(k.WithNamespace(IPv6Namespace).
+							ApplyKustomize("tcp-echo-ipv6")).
+							To(Succeed(), "error deploying ipv6 pod")
+						Expect(k.WithNamespace(SleepNamespace).
+							ApplyKustomize("sleep")).
+							To(Succeed(), "error deploying sleep pod")
```

Comment:
```suggestion
						Expect(k.WithNamespace(DualStackNamespace).ApplyKustomize("tcp-echo-dual-stack")).To(Succeed(), "error deploying tcpDualStack pod")
						Expect(k.WithNamespace(IPv4Namespace).ApplyKustomize("tcp-echo-ipv4")).To(Succeed(), "error deploying ipv4 pod")
						Expect(k.WithNamespace(IPv6Namespace).ApplyKustomize("tcp-echo-ipv6")).To(Succeed(), "error deploying ipv6 pod")
						Expect(k.WithNamespace(SleepNamespace).ApplyKustomize("sleep")).To(Succeed(), "error deploying sleep pod")
```

Nit, I think it's fine being one liners, but it's a matter of taste ;)

Reply:
I changed because the lint was complaining about the length of the line. Our linter complains when a certain line takes more than X characters. I'll do the change and rollback if complaints again